src/axeResultToEarl.ts

```diff
@@ -32,6 +32,12 @@ export default function axeResultToEarl(
           const version = (match && match[1]) || ''
           return {
             '@type': EarlType.Assertion,
+            'doap:name': 'Axe',
+            'doap:vendor': {
+              '@id': 'https://deque.com/',
+              '@type': 'foaf:Organization',
+              'foaf:name': 'Deque Systems'
+            },
```

Comment:
These don't belong on the assertion. If you want to include info about the tool directly into the output data you need put an Assertor object in the assertedBy property, see: https://www.w3.org/TR/EARL10-Schema/#Assertor